docs/conf.py

```diff
@@ -62,4 +64,4 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-# html_static_path = ["static"]
+html_static_path = ["_static"]
```

Comment:
Aren't the files still in` static`? How do they end up in `_static` now?

Reply:
We can probably drop this.  However, sphinx copies the theme "static" to the html "_static" by default so far as I can tell...

Reply:
I dropped this change - it probably came around as I was trying to understand what was going on in the pydata-sphinx-theme.